apps/client/src/components/criminals/criminals.module.css

```diff
@@ -7,14 +7,17 @@
   height: 100%;
 }
 .criminalsContainer {
-  overflow: scroll;
+  overflow-y: scroll;
   display: grid;
-  grid-template-columns: auto auto;
-  grid-template-rows: auto auto;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: repeat(2, 1fr);
   gap: 0.5em;
   padding: 0.5em;
   height: 90%;
+  scrollbar-width: thin;
+  scrollbar-color: var(---red-border-color) #ffffff;
```

Comment:
check the issue with the scroll bar not centered around the white scrollbar background